pkg/crypto/pedersen/pedersen.go

```diff
@@ -8,6 +8,48 @@ import (
 	"math/big"
 )
 
+// See https://github.com/dontpanicdao/caigo/blob/e34006317632c87334fa29e770a00b5c7c94cb20/curve.go#L339-L351
+
+// divMod finds a nonnegative integer x < p such that (m * x) % p == n.
+// Assumes that m and p are coprime. This implementation is only meant to be used
+// in `pedersen.add`, where this assumption holds.
```

Comment:
Wrap comment lines at 72 columns where possible i.e. with the exception of URLs.